### ISSUE TITLE
Figure 1.16 Wrong

### ISSUE THREAD

**Issue by ncbaratta** [PacktPublishing/The-Python-Workshop-Second-Edition#2]
Figure 1.16 on page 36 is wrong.

**Comment by coreyjwade** [comment]
Yes, it's too bad this was missed. I'm a math teacher, and mathematically the symbols have no intrinsic meaning as you can read it either way. But in programming they have a clear meaning and it has been flipped.
